starters/features/bootstrap/src/components/bootstrap/navbar.tsx

```diff
@@ -0,0 +1,47 @@
+import { component$, useVisibleTask$ } from "@builder.io/qwik";
+import { Link } from "@builder.io/qwik-city";
+export const Navbar = component$(() => {
+  useVisibleTask$(async () => {
```

Comment:
I think this might be less than ideal. Not sure what's the right course of action, but it seems it might be a good idea to import the entire library instance for this to work.

Thought here @gioboa @mhevery ?

Reply:
Hi (I'm mugan86 from another account),

What do you mean by: "I'm not sure what the correct course of action is, but it seems like it might be a good idea to import the entire library instance to make this work."

If you are referring to the "bootstrap" import, this is used so that we already have the Javascript functionalities, in this case so that if we have our application in responsive, the toggle menu of that navbar can be opened or not. Without this import element, Boostrap could only be used at the style level, but not at the functionality level.

I was thinking about putting it in the src/bootstrap/layout.tsx file but I am waiting for your suggestions for improvement and accordingly.

Perhaps I have not understood you well, I am waiting for you to tell me what would be the most correct way to implement these details.

I hope you see the rest as correct and if not, obviously let me know so I can leave it as you recommend your team.

I have already been improving many details for the procedure thanks to the suggestions that @gioboa made in a previous PR ;)

Thank you so much

Reply:
Hi @anartzdev !
First off, thanks for the PR! (forgot to say that before 😅 )

I was chatting with @gioboa and we are still mulling it over. What we're currently thinking is that it might be better to avoid using `useVisableTask` here and loading the bootstrap script via an HTML script tag, via the `root.tsx`.

Reply:
@gioboa mentioned another option:
something like this --> https://github.com/BuilderIO/qwik/blob/b330651d33bfa9665f1619e29c831e69286c11a0/starters/apps/basic/src/components/router-head/router-head.tsx#L32
```
export const RouterHead = component$(() => {
  [...]
  return (
    <>
      [...]
      {head.scripts.map((s) => (
        <script key={s.key} {...s.props} dangerouslySetInnerHTML={s.script} />
      ))}
    </>
  );
});
```

But we both think `root.tsx` is fine.

Reply:
So `useVisibleTask$` causes code to execute eagerly, which is why we try to push back (especially in libraries) on its use...

The other point about `useVisibleTask$` is that it causes the FW to be loaded on event....

1. So the best option would be to do nothing, which sounds like is not possible in this case as we do need to load the `bootstrap`
2. If it would be used everywhere, I would say put it in global, but in this case, it is only specific to the component, so visible is appropriate. So I would use `useOn('qvisible', $(...))` as this code will nat cause the framework to load when the component becomes visible. 
3. The option of last resort is `useVisibleTask$`, which we are trying to avoid because it will cause the FW to be loaded as well...

So I think the correct thing here is to switch to `useOn('qvisible', $(...))`

PS: Thank you for the hard work of creating this PR.

Reply:
Thank you very much for your answers and feedback. All the points you are presenting to me are really interesting and serve as a learning experience for me. The truth is that by creating some things to contribute I am learning more than by diving into the documentation.

So, before resorting to useVisibleTask$ (which should already be the "desperate" situation when there is no other option) we are going to resort to useOn('qvisible', $(...)).

Very interesting, I make that change and when I have it, I'll let you know.

Thank you very much team, to @hamatoyogi, @gioboa and @mhevery. A pleasure to contribute to this great project and I hope to continue doing so! I have some ideas in mind, which I will propose, as I learn from you :)